packages/app-mobile/package.json

```diff
@@ -79,6 +80,7 @@
     "stream": "0.0.2",
     "stream-browserify": "3.0.0",
     "string-natural-compare": "3.0.1",
+    "tar-stream": "3.1.6",
```

Comment:
The `tar` package depends on `fs`, but `tar-stream` can work without it. Thus, `tar-stream` is used here.